group.py

```diff
@@ -2,4 +2,21 @@
 
 # Your code to go here...
 
-my_group =
+my_group = {
+		'Jill' :{
+			'age': 26,
+			'job':['biologist'],
+			'relations':{'friend':['Zalika'],'partner':['John']}
+			},
+                'Zalika' :{
+			'age':28,
+			'job':['Artist'],
+			'relations':{'friend':['Jill']}
+			 }, 
+		'John' :{
+			'age':27,
+                        'job':['Writer'],
+                        'relations':{'partner':['Jill']
```

Comment:
You have added in reciprocal relationships between friends and partners,  what about other relationships?